### PR TITLE
Update documentation for Command files

### LINKED ISSUE #133: Check Command of Logic folder for any documentation bugs
delete commented out code
check if any missing javadocs/grammar starts correctly
check for unused variable, etc

### PR DESCRIPTION
**Pull request by whatthelump**
Fixes #133. Note that there are still commented out tests for EditCommand and StatusCommand, not sure if we should reinstate them or just delete them?

### CODE REVIEW

**Comment by VRSoorya** [comment]
Regarding the tests maybe we could keep them commented and re-evaluate in v1.4?

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 1]
```diff
@@ -45,18 +39,13 @@
...
-    public static final String BOOKING_DETAILS_DESC_RESIDENCE2 = " " + PREFIX_BOOKING + VALID_BOOKING_DETAILS;
+    public static final String VALID_BOOKING_START1 = "03-03-2021";
+    public static final String VALID_BOOKING_START2 = "30-03-2021";
+    public static final String VALID_BOOKING_END1 = "05-05-2021";
+    public static final String VALID_BOOKING_END2 = "30-05-2021";
+
     public static final String NAME_DESC_RESIDENCE1 = " " + PREFIX_NAME + VALID_NAME_RESIDENCE1;
     public static final String NAME_DESC_RESIDENCE2 = " " + PREFIX_NAME + VALID_NAME_RESIDENCE2;
-    public static final String PHONE_DESC_AMY = " " + PREFIX_PHONE + VALID_PHONE_AMY;
-    public static final String PHONE_DESC_BOB = " " + PREFIX_PHONE + VALID_PHONE_BOB;
```
Comment (VRSoorya) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:59]: I think we need these to make bookings to be used in tests later but maybe could rename to PHONE_DESC_BOOKING1 and PHONE_DESC_BOOKING2?

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -32,10 +30,6 @@
     public static final String VALID_ADDRESS_RESIDENCE1 = "14 Amber Gardens, 439960";
     public static final String VALID_ADDRESS_RESIDENCE2 = "Block 50, Cantonment Rd";
 
-    public static final String VALID_BOOKING_DETAILS = "4 Adults";
-    //public static final List<Booking> VALID_BOOKING_LIST1 = new BookingList()
-    //        .add(new Booking(VALID_NAME_AMY, VALID_PHONE_AMY, VALID_BOOKING_START1, VALID_BOOKING_END1));
-    //public static final List<Booking> VALID_BOOKING_LIST2 = "2 Teenagers";
```
Comment (VRSoorya) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:38]: I left this commented out to be built on later when we add more tests so could we keep sorry!

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 3]
```diff
@@ -45,18 +39,13 @@
 
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
-    public static final String VALID_NAME_AMY = "Amy";
```
Comment (VRSoorya) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:48]: probably need to keep this line too to make bookings in tests later?

**Review by VRSoorya — COMMENTED** [review]
LGTM! Just some minor requests though.

**Review by VRSoorya — APPROVED** [review]

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 1 above)
Comment (whatthelump) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:59]: This sounds sensible, I have replaced references to "Amy" and "Bob" with "Booking1" and "Booking2" respectively.

**Pull request merged**